playbooks/roles/mongo_3_2/tasks/main.yml

```diff
@@ -192,10 +205,45 @@
     src: mongo_logrotate.j2
     dest: /etc/logrotate.d/hourly/mongo
   tags:
+    - "backup:mongo"
     - "install"
     - "install:app-configuration"
     - "logrotate"
 
+- name: install backup script
+  template:
+    src: backup-mongo.sh.j2
+    dest: "{{ mongo_backup_script_path }}"
+    mode: 0700
+  tags:
+    - "backup:mongo"
+    - "install"
+
+- name: add mongo backup script to cron
+  cron:
+    name: mongo backup job
+    minute: "{{ mongo_backup_cron.minute | default('12') }}"
+    hour: "{{ mongo_backup_cron.hour | default('*/12') }}"
```

Comment:
@swalladge I'm only seeing one snapshot per day for prod-olivex-mongo-backup in [this list](https://eu-west-1.console.aws.amazon.com/ec2/v2/home?region=eu-west-1#Snapshots:visibility=owned-by-me;description=mongo;sort=desc:startTime). According to this definition there should be two? (This is the case for stage-olivex-mongo-backup.) You mentioned [here](https://tasks.opencraft.com/browse/SE-1670?focusedCommentId=131361&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-131361) that the backup process took 7h to run, so with a cadence of 1 snapshot every 12 hours, the process of creating the first snapshot for a given day shouldn't interfere with the process for creating the second one?

Reply:
The backups were only enabled yesterday, so that's expected. I'm seeing a new snapshot now that's 12h later than the previous snapshot. We can wait a couple of days to make sure though. :+1:

Reply:
OK cool, I'm seeing two snapshots for yesterday as well. Since this has been working reliably for stage I think it's fine to assume that the same is true for prod 👍